code/game/gamemodes/traitor/king_disk.dm

```diff
@@ -35,3 +35,25 @@
 	escape_objective.owner = traitor
 	traitor.objectives += escape_objective
 	return
+
+/datum/game_mode/traitor/king_disk/process()
+	for(var/obj/item/weapon/disk/nuclear/N in poi_list)
+		var/atom/disk_loc2 = N.loc
+		if(istype(disk_loc2, /mob))
+			var/mob/M = disk_loc2
+			var/datum/mind/traitor = M.mind
```

Comment:
You might want to make sure the mob has a mind first, or you will run into some nasty runtimes.